samples/started-guides/README.md

```diff
@@ -0,0 +1,29 @@
+# Indy SDK getting started guide
```

Comment:
I suggest to don't use "Getting Started Guide" title as we already have "Getting Started Guide" as a dedicated artifact